src/content/ui-api/web/organizations/access-control/index.md

```diff
@@ -0,0 +1,175 @@
+
+- Show all roles with `read` in the name.
+- Show all roles where `someone@example.com` appears as a subject.
+- Show all roles explicitly granting permission to `events`.
+
+## Granting access {#granting-access}
+
+Granting access to an organization's resources means, in Kubernetes RBAC terms: you assign a role or cluster role to the selected organization's namespace, by creating role bindings in this namespace.
+
+In order to assign a role to a user, group, or service account, follow these steps:
+
+### 1. Select a role
+
+Find and select the role or cluster role to assign within the list on the left hand side. Select the role that defines the permissions you want to grant.
+
+For example, the `cluster-admin` ClusterRole, when assigned here, will grant all permissions within this organization's namespace. The `read-all` ClusterRole instead gives read-only access to all resources in this namespace.
+
+### 2. Double-check permissions
+
+By inspecting the _Permissions_ tab, double check that the role grants only the permissions you are expecting.
+
+### 3. Add subjects
+
+On the _Subjects_ tab, add the group(s) or user(s) you want to assign to the selected role.
+
+When adding a **group** or a **user**, you have to enter the exact user or group identifier as your identity provider presents them. This includes upper and lowercase spelling. User identifiers may or may not be email addresses, according to your set-up.
+
+**Note:** Your Giant Swarm account engineer (AE) is happy to assist in case there are uncertainties about the correct spelling.
+
+You can enter or paste multiple identifiers, separated by blank or comma, into a single input field.
+
+When adding a **service account** as a subject, you are free to select from the existing ones in the current organization's namespace, by using the suggestions in the input field. When typing some letters, these suggestions will present existing service accounts that match your input. However you can also create new service accounts here by entering new names here.
+
+In either case, confirm by clicking the _OK_ button.
+
+Done. You have successfully assigned subjects to one specific role for one specific organization.
+
+### Resulting resources
+
+As a result of the action above, there will be new role bindings (`RoleBinding` resources) in the organization's namespace. These resources follow the naming convention
+
+```nohighlight
+<role-name>-<timestamp>
+```
+
+As an example, a `RoleBinding` for a `Role` or `ClusterRole` named `read-all` could be named `read-all-1618868398846`.
+
+As mentioned, new service accounts may have been created in the organization's namespace.
+
+You are free to edit or delete the role bindings and service accounts created via the web user interface in some other way via the management API. Treat them as if they were yours.
+
+## Revoking access
```

Comment:
I am putting myself on customer shoes and I see they would like to remove a user from a certain group. I know this means removing the user from IdP but I would add a warning/note here explaining the case

Reply:
Maybe the same for the addition of a certain user to a group in IdP (like "adding a user in the same user group on your IdP connector will result in assigning permissions automatically without any further action in our platform")

Reply:
Sounds reasonable, will add something.